cmd/bootnode/main.go

```diff
@@ -69,6 +83,10 @@ func main() {
 			log.Crit("unable to convert decoded hex into ecdsa.PrivateKey", "data", key, "error", err)
 		}
 	}
+	if *writeAddr {
+		fmt.Printf("%x\n", crypto.FromECDSAPub(&nodeKey.PublicKey)[1:])
```

Comment:
Why omitting the first byte? It's that compressed/uncompressed info?

Reply:
I just copied it from `go-ethereum` repo:
https://github.com/ethereum/go-ethereum/blob/c2d65d34/cmd/bootnode/main.go#L90-L93
Any idea why they did it like that?

Reply:
Oh, I think they are just removing the `0x` from it so it's ready to be used in an `enode://` address.

Reply:
No, it's a bytes slice. `0x` is added when formatting to a string. It's about that uncompressed byte: https://github.com/golang/go/blob/go1.13.1/src/crypto/elliptic/elliptic.go#L310. LGTM but we can add a comment why we do `[1:]`.

Reply:
Okay, I added a comment explaining it. Okay to merge?